src/astra_driver.cpp

```diff
@@ -526,14 +526,14 @@ void AstraDriver::configCb(Config &config, uint32_t level)
   data_skip_ = config.data_skip+1;
 
   // Call Stream activators whenever the always_active parameters are changed
-  if (config.always_active_rgb_stream != always_active_rgb_stream_)
+  if (config_init_ && config.always_active_rgb_stream != always_active_rgb_stream_)
```

Comment:
Don't you still want to call the callbacks in the first run? If yes, both blocks should be moved [here](https://github.com/magazino/ros_astra_camera/pull/3/files#diff-d86fae028e607c33f405905d4f3fa062c96808e746fb9b9510abfc182d9abbb5L547).